.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,10 +23,10 @@
 <!-- check the items below that will be completed prior to merge.
      strikethrough any item text that does not apply to this PR. -->
 
-- [ ] :ok_hand: design updates will be Garden Designer approved (add the designer as a reviewer)
+- [ ] :ok_hand: design updates are Garden Designer approved (add the designer as a reviewer)
 - [ ] :globe_with_meridians: demo is up-to-date (`yarn start`)
 - [ ] :arrow_left: renders as expected with reversed (RTL) direction
 - [ ] :metal: renders as expected with Bedrock CSS (`?bedrock`)
-- [ ] :guardsman: includes new unit tests
-- [ ] :wheelchair: tested for [WCAG 2.1](https://www.w3.org/TR/WCAG21) AA compliance
+- [ ] :guardsman: includes new unit tests. Maintain [existing coverage](https://coveralls.io/github/zendeskgarden/react-components?branch=main) (always >= 94%)
```

Comment:
```suggestion
- [ ] :guardsman: includes new unit tests. Maintain [existing coverage](https://coveralls.io/github/zendeskgarden/react-components?branch=main) (always >= 96%)
```

Nice. I think we can inflate to >= 96%, which we've worked hard to achieve here.

Reply:
upped